docs/generated/packages/angular/generators/application.json

```diff
@@ -43,13 +45,15 @@
               "label": "LESS         [ http://lesscss.org     ]"
             }
           ]
-        }
+        },
+        "x-priority": "important"
```

Comment:
I think `style` should not be important since it will be read from `nx.json` most of the time.

Reply:
should be fixed up now